core/models.py

```diff
@@ -127,6 +128,7 @@ class Languages(models.TextChoices):
     )
     activated = models.BooleanField(null=True, default=False)
     deleted = models.BooleanField(default=False)
+    bundle_analysis_enabled = models.BooleanField(null=True, default=False)
```

Comment:
What would be the use case for a null value here?

Reply:
Good point, there is no case, removed it. I just copy pasted the coverage activated field.